sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_container_client.hpp

```diff
@@ -324,6 +310,19 @@ namespace Azure { namespace Storage { namespace Blobs {
     Azure::Core::Response<Models::BreakBlobContainerLeaseResult> BreakLease(
         const BreakBlobContainerLeaseOptions& options = BreakBlobContainerLeaseOptions()) const;
 
+    /**
+     * @brief Marks the specified blob or snapshot for deletion. The blob is later deleted
+     * during garbage collection. Note that in order to delete a blob, you must delete all of its
+     * snapshots. You can delete both at the same time using DeleteBlobOptions.DeleteSnapshots.
```

Comment:
As per recent research, I noticed you can use `#` to reference the API like
```suggestion
     * snapshots. You can delete both at the same time using #Azur::Storage::Blobs::DeleteBlobOptions::DeleteSnapshots.
```

Reply:
Nit: there is a typo in the suggested change: `#Azure`

Reply:
@vhvb1989 I will refine the doc in the future (after GA). I opened an issue to track that. I will keep these things unchanged for now to make this PR single-purpose.